packages/template-retail-react-app/app/pages/account/index.test.js

```diff
@@ -86,50 +86,45 @@ describe('Test redirects', function () {
         )
     })
     test('Redirects to login page if the customer is not logged in', async () => {
-        sdk.useCustomerType.mockReturnValue({isRegistered: false, isGuest: true})
-        const Component = () => {
```

Comment:
The MockComponent has included this route to render account page, we don't need these lines